js/index.js

```diff
@@ -1,24 +1,36 @@
 // ITERATION 1
-
 function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
 
-  //... your code goes here
+  const price = product.querySelector('.price span');
+  const quantity = product.querySelector('.quantity span');
+let priceValue = price.innerHTML;
+let quantityValue = quantity.innerHTML;
+let subtotalPrice = priceValue*quantityValue;
+
+document.getElementsByClassName('subtotal').getElementByTagName('span').innerHTML= subtotalPrice;
+return subtotalPrice;
 }
 
+ // ITERATION 2
+  //... your code goes here
 function calculateAll() {
   // code in the following two lines is added just for testing purposes.
   // it runs when only iteration 1 is completed. at later point, it can be removed.
-  const singleProduct = document.querySelector('.product');
-  updateSubtotal(singleProduct);
+  //const singleProduct = document.querySelector('.product');
+  //updateSubtotal(singleProduct);
   // end of test
+ let product = document.getElementsByClassName('product');
```

Comment:
getElementsByClassName is also returning an array, A way of solving is using a for loop